user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementDetailsComponent.java

```diff
@@ -188,10 +190,19 @@ private void createProteomicsGrid() {
     proteomicsMeasurementGrid.addClassName("measurement-grid");
     proteomicsMeasurementGrid.addColumn(
             proteomicsMeasurement -> proteomicsMeasurement.measurementCode().value())
-        .setHeader("Measurement Code").setAutoWidth(true).setTooltipGenerator(proteomicsMeasurement -> proteomicsMeasurement.measurementCode().value());
+        .setHeader("Measurement Code").setAutoWidth(true).setTooltipGenerator(
+            proteomicsMeasurement -> proteomicsMeasurement.measurementCode().value()).setFlexGrow(0);
```

Comment:
for my understanding; not a change request; what is the `setFlexGrow(0)` here? Why is it necessary?

Reply:
The Flexgrow call ensures that the this column width doesn't grow or shrink if the grid size changes. This was done to ensure that the measurement code is always readable. For more detail see:
https://vaadin.com/docs/latest/components/grid#columns

Reply:
thank you for the clarification! :)